WordPress/Classes/ViewRelated/Site Creation/Design Selection/SiteDesignContentCollectionViewController.swift

```diff
@@ -57,12 +57,15 @@ class SiteDesignContentCollectionViewController: FilterableCategoriesViewControl
 
     init(_ completion: @escaping SiteDesignStep.SiteDesignSelection) {
         self.completion = completion
+        let isLastSiteCreationStep = ABTest.siteNameV1.variation == .treatment(nil) && FeatureFlag.siteName.enabled
+        let primaryButtonTitle = isLastSiteCreationStep ? NSLocalizedString("Create site", comment: "Title for the button to progress with creating the site with the selected design")
+            : NSLocalizedString("Choose", comment: "Title for the button to progress with the selected site homepage design")
 
         super.init(
             analyticsLocation: "site_creation",
             mainTitle: NSLocalizedString("Choose a design", comment: "Title for the screen to pick a design and homepage for a site."),
             prompt: NSLocalizedString("Pick your favorite homepage layout. You can edit and customize it later.", comment: "Prompt for the screen to pick a design and homepage for a site."),
-            primaryActionTitle: NSLocalizedString("Choose", comment: "Title for the button to progress with the selected site homepage design"),
```

Comment:
similar comments here: the comment should have a period, and since this file has quite a few hardcoded strings, wondering if we can group them?

Reply:
As above handled with babc542445c63b38afa12643abe19746e3dc30cb. Thank you for the feedback 🙇